fbs.info

```diff
@@ -2,11 +2,14 @@ name = FBS
 description = Provider for FBS.
 core = 7.x
 package = Providers
+version = 7.x-1.0-dev
+project = fbs
 dependencies[] = date_popup
 dependencies[] = ding_provider
 dependencies[] = ding_reservation
 dependencies[] = ding_user
 dependencies[] = entity
+dependencies[] = fbs
```

Comment:
Ask features. I guess it gets added because the feature uses fields from the module, and Features isn't smart enough to notice.